_Sidebar.md

```diff
@@ -15,11 +15,13 @@
 12. **[[Interoperability|Interoperability]]**
 13. **[[Kokkos and Virtual Functions|Kokkos and Virtual Functions]]**
 ### **[[Content: Containers|ContentContainers]]**
-1. **[DualView](Kokkos%3A%3ADualView)**
-2. **[DynRankView](Kokkos%3A%3ADynRankView)**
-3. **[DynamicView](Kokkos%3A%3ADynamicView)**
-4. **[[OffsetView|Offset-View]]**
-5. **[[UnorderedMap|Unordered-Map]]**
+1. **[[Bitset|Bitset]]**
```

Comment:
Note this is the programming guide link. This should not ahve the links to the API Reference. I think the links for most of the other classes here are wrong too.

Reply:
I'll move all the containers from programming guide section of the sidebar to the API reference section.